### ISSUE TITLE
QupZilla 1.8.8 - AwOken Iconset Home Button Missing

### ISSUE THREAD

**Issue by Geyup** [QupZilla/qupzilla#1755]
In Linux, using the AwOken 2.5 Iconset;

http://alecive.deviantart.com/art/AwOken-163570862

The Home button doesn't appear in 1.8.8. This did work ok in 1.8.6

What names does QupZilla use to load a HOME icon?

![qupzilla](https://cloud.githubusercontent.com/assets/13072581/11025137/7a7e774c-868f-11e5-9134-ed2a5f608699.jpg)

**Comment by cranes-bill** [comment]
`go-home`

**Comment by cranes-bill** [comment]
@Geyup , **the following it's for the developers, just to make this thread more sense:** :stuck_out_tongue_winking_eye:
Trying to download this icon theme, i'm getting the following error:
```sh
WebPage::UnsupportedContent  QUrl( "http://packages.slackonly.com/pub/packages/current-x86_64/desktop/awoken-icon-theme/awoken-icon-theme-2.5-noarch-1_slack.txz" )  ProtocolUnknowError 
WebPage::UnsupportedContent  QUrl( "http://packages.slackonly.com/pub/packages/current-x86_64/desktop/awoken-icon-theme/awoken-icon-theme-2.5-noarch-1_slack.txz" )  ProtocolUnknowError 
WebPage::desktopServicesOpen Url  QUrl( "http://packages.slackonly.com/pub/packages/current-x86_64/desktop/awoken-icon-theme/awoken-icon-theme-2.5-noarch-1_slack.txz" )  has already been opened!
Ignoring it to prevent infinite loop!
```
The above it's automatically repeating.

**Comment by nowrep** [comment]
@Geyup What DE are you using?
@cranes-bill Well, that's fine unless it doesn't stop.

**Comment by cranes-bill** [comment]
It stops, but does not downloading the archive!
About the icons, this theme does not contain _go-home_ icon...

**Comment by nowrep** [comment]
If the theme doesn't have `go-icon` then you should report the bug to the authors. It is standard icon (http://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html)

**Comment by Geyup** [comment]
I'm using Openbox and I use LXAppearance to load themes and iconsets...

I have go-home.png, this appears to be a problem for some reason with QupZilla not loading it...

This worked just fine in previous versions of QupZilla.

Here's the iconset to please test out;

http://alecive.deviantart.com/art/AwOken-163570862

**Comment by Geyup** [comment]
Actually after replying, I compiled 1.8.9 and it now works. Hmm seems to of been some bug in 1.8.8

So all good now... :)

**Comment by nowrep** [comment]
That's very unlikely, but I'm glad it works for you now.

**Comment by Geyup** [comment]
Actually I forgot about mentioning QT4, I just realized this does not work in 1.8.6, 1.8.7, 1.8.8 and 1.8.9 using QT4, but it does work in QT5.

The icon is named ```go-home``` I looked at the icon, and compared it to other iconsets, the permissions were the same, the only difference was the extension, others were svg and this is png.

If I load another iconset with the same name it works.

I don't understand how this would not work in QT4 with one iconset, but work with others... 

hmm
